contracts/BillingConnector.sol

```diff
@@ -136,6 +136,10 @@ contract BillingConnector is IBillingConnector, Governed, Rescuable, L1ArbitrumM
     ) external payable override {
         require(_amount != 0, "Must remove more than 0");
         require(_to != address(0), "destination != 0");
+        // Callers of this function should generally be L1 contracts
+        // (e.g. multisigs) that don't exist in L2, so the destination
+        // must be some other address.
+        require(_to != msg.sender, "destination != sender");
```

Comment:
```suggestion
        require(_to != msg.sender || !isContract(_to), "destination != sender or not a contract");
```

Probably not worth the extra import and effort but this would solve for the extremely specific case of an EOA on L1 wanting to remove to L2 without having to interact with the billing contract on L2.

Reply:
We had a quick chat about this on Slack with the OZ team, and the suggestion was to not do this so that we push users to use the correct flow. I was hesitant but in the end I don't see why a user would want to do this instead of just going straight to L2, so I'd say let's leave it like this to reduce surface.